code/modules/antagonists/monsterhunter/monstertrack.dm

```diff
@@ -43,7 +48,7 @@
 			monsters += carbon_minds
 		if(carbon_minds.has_antag_datum(/datum/antagonist/ashwalker))
 			monsters += carbon_minds
-		if(carbon_minds.has_antag_datum(/datum/antagonist/wizard/apprentice))
```

Comment:
why dab on apprentices

Reply:
has_antag_datum checks subtypes if you don't set check_subtypes to false, so we don't need to check for new bloods or wizard apprentices as they are subtypes of vampire and wizard respectively.